package.json

```diff
@@ -72,6 +73,7 @@
     "pre-commit": "lint-staged"
   },
   "lint-staged": {
+    "*": "prettier --ignore-unknown --write",
```

Comment:
note: we might want to check if these lint-staged are running parallelly or serially. Sometimes the other command can override these results as well.

Reply:
You're right, https://github.com/lint-staged/lint-staged?tab=readme-ov-file#task-concurrency lists this as a direct example. I'll touch this up.